### ISSUE TITLE
BBC Micro emulators

### ISSUE THREAD

**Issue by diegop77** [petrockblog/RetroPie-Setup#583]
This is the link of the project that have a version for linux of the BBC- Micro computer, can you add this on RetroPie?

Link: http://b-em.bbcmicro.com/

Thank you.

Diego

**Comment by alexxxmg** [comment]
Having just installed RetroPie - and having a lot of fun with it - I was surprised not finding the emulation of the BBC (just considering some really obscure systems it is able to emulate!)
I cast my vote for it being supported!!!

**Comment by objectchris** [comment]
looks like the full installation process is detailed here http://haprpi.blogspot.co.uk/p/the-bbc-lives.html?m=1 - looks like it should be possible!

**Comment by alexxxmg** [comment]
thank you objectchris!

**Comment by oneswitch** [comment]
Agreed. Would love for there to be an easy default installation of a quality BBC Micro emulator for RetroPie. That would be fantastic for many reasons. The best emulator on Windows I've found is BeebEm: http://www.mkw.me.uk/beebem/ - and there's a UNIX/Linuz version here: http://beebem-unix.bbcmicro.com/

**Comment by Marshallx** [comment]
wow bbc micro not included by default, what gives? +1

**Comment by linky00** [comment]
Jesus man, I wanna play Elite! I'd love to see a default BBC Micro emulator!

**Comment by methanoid** [comment]
Periodically I google to see if this exists.... Does nobody care about the BBC Micro for emulation? I just assumed Retropie would have it! :-(

**Comment by nmstoker** [comment]
The instructions linked to by @dankcushions work well (http://haprpi.blogspot.co.uk/p/the-bbc-lives.html?m=1 ) it just needs a bit of polish
Haven't got sound working yet, but will try that over the holidays
